### DISCUSSION TITLE
How to specify source path properly

### DISCUSSION THREAD

**Discussion by marsupialtail** [mkdocstrings/mkdocstrings#539] [Q&A] [answered]
This is an example of what I have inside of a markdown: 

```
# DataStream.filter

::: pyquokka.datastream.DataStream.filter

```

This seems to automatically go get the latest code from the github repo, but not in the local source directory.

So my question -- how to specify local source code directory for mkdocstrings? Sorry for simple question

**Comment by pawamoy** [comment, marked as answer]
Hello! mkdocstrings definitely does not fetch anything from GitHub, it uses only local files. Happy to investigate any issue if you can provide a reproducible project :slightly_smiling_face:
